### ISSUE TITLE
Last step: /etc/bluetooth/main.conf

### ISSUE THREAD

**Issue by backupandforkacc** [fdanis-oss/pw_wp_bluetooth_rpi_speaker#6]
I try to install the whole setup on ubuntu. I do not understand the last step. Can I just add some lines to the file  /etc/bluetooth/main.conf?

What expression exactly do I have to add here?

**Comment by fdanis-oss** [comment]
Yes, you can add the `JustWorksRepairing` option directly to the `/etc/bluetooth/main.conf`.

You can do this by running:
```
$  echo "JustWorksRepairing = always" | sudo tee /etc/bluetooth/main.conf
```

**Comment by flavi1** [comment]
Hello, since it is probably relative to the same configuration file, i don't open a new ticket.
 Is it possible to add an option to define a code PIN?
If I connect a smartphone, the code PIN defined on the raspberry pi will be asked on the phone ?
So my neighbors will not be able to play music in my room anymore :o)

**Comment by fdanis-oss** [comment]
@flavi1 The PIN code is not managed in the BlueZ configuration file but in the authentication/authorization agent.\
You can find a sample of the speaker-agent.py with an hard-coded PIN in 69ed5f9 ([add_pin_code](https://github.com/fdanis-oss/pw_wp_bluetooth_rpi_speaker/tree/add_pin_code) branch).

**Comment by flavi1** [comment]
Excellent! Thank you
